C7FIT/ScheduleBrowserTableViewCell.swift

```diff
@@ -0,0 +1,75 @@
+//
+//  ScheduleBrowserTableViewCell.swift
+//  C7FIT
+//
+//  Created by Michael Lee on 1/25/17.
+//  Copyright © 2017 Brandon Lee. All rights reserved.
+//
+
+import UIKit
+
+class ScheduleBrowserTableViewCell: UITableViewCell {
+
+    // MARK: - Properties
+    
+    lazy var scheduleTitle: UILabel = UILabel()
+    lazy var schedulePicture: UIImageView = UIImageView()
+    lazy var scheduleLink: UIButton = UIButton()
+
+    // MARK: - Initialization
+
+    override init(style: UITableViewCellStyle, reuseIdentifier: String?) {
+        super.init(style: style, reuseIdentifier: reuseIdentifier)
+        print("browser cell")
+        setup()
+        setupConstraints()
+    }
+    
+    required init?(coder aDecoder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+    
+    // MARK: - Layout
+    
+    func setup() {
+        
+        let imageName = "club front.jpg"
+        let bgImage = UIImage(named: imageName)
+
+        
+        if (bgImage != nil) {
+            schedulePicture.image = bgImage!
+            schedulePicture.center = self.center
+
+            addSubview(schedulePicture)
+            print("image success")
+        } else {
+            print("image error")
+        }
+        
+        scheduleLink.setImage(bgImage, for: .normal)
+        addSubview(scheduleLink)
+        
+        scheduleTitle.text = "Monthly Schedule"
```

Comment:
would be good practice to wrap all user-facing strings in NSLocalizedString(key, comment) to all multiple languages someday